src/hyperlight_common/src/flatbuffer_wrappers/function_call.rs

```diff
@@ -72,91 +71,29 @@ impl FunctionCall {
     pub fn function_call_type(&self) -> FunctionCallType {
         self.function_call_type.clone()
     }
-}
-
-#[cfg_attr(feature = "tracing", instrument(err(Debug), skip_all, parent = Span::current(), level= "Trace"))]
-pub fn validate_guest_function_call_buffer(function_call_buffer: &[u8]) -> Result<()> {
-    let guest_function_call_fb = size_prefixed_root::<FbFunctionCall>(function_call_buffer)
-        .map_err(|e| anyhow::anyhow!("Error reading function call buffer: {:?}", e))?;
-    match guest_function_call_fb.function_call_type() {
-        FbFunctionCallType::guest => Ok(()),
-        other => {
-            bail!("Invalid function call type: {:?}", other);
-        }
-    }
-}
 
-#[cfg_attr(feature = "tracing", instrument(err(Debug), skip_all, parent = Span::current(), level= "Trace"))]
-pub fn validate_host_function_call_buffer(function_call_buffer: &[u8]) -> Result<()> {
```

Comment:
Maybe I missed it, but where was this function called?
If it wasn't used at all, I am curious how the compiler hasn't warned us.